docs/CONFIGURATION.md

```
@@ -0,0 +1,308 @@
+## Configuration File Format
+
+Project configuration files use TOML format for clarity and flexibility:
+
+```toml
+# .wtconfig example
+
+[global]
+default_branch = "main"
+default_editor = "cursor"
+auto_open = true
+
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)